.github/workflows/cron.yml

```diff
@@ -1,41 +1,90 @@
-# name: Cron
-
-# on:
-#   schedule:
-#     # should run every 3hrs on weekdays
-#     - cron: "0 */3 * * 1-5"
-#     # for testing purposes (this will run every 5mins everyday)
-#     # - cron: "*/5 * * * *"
-
-# jobs:
-#   notify:
-#     # run slack-notifier workflow
-#     uses: ./.github/workflows/call-slack-notifier.yml
-#     with:
-#       message: Starting scheduled e2e tests run...
-#     secrets:
-#       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-
-#   tests:
-#     runs-on: ubuntu-latest
-
-#     steps:
-#       - name: check out branch
-#         uses: actions/checkout@v3
-
-#       - name: Grab Node 16.x
-#         uses: actions/setup-node@v3
-#         with:
-#           node-version: '16.x'
-#           cache: 'npm'
-#           cache-dependency-path: '**/package-lock.json'
-
-#       - name: Npm ci
-#         run: npm ci
-
-#       - name: Run Playwright tests
-#         run: npm run test
-
-#       - name: Publish test report
-#         uses: ./.github/workflows/call-publish-report.yml
+name: Cron
 
+on:
+  schedule:
+    # should run every 3hrs on weekdays
+    - cron: "0 */3 * * 1-5"
+    # for testing purposes (this will run every 5mins everyday)
+    # - cron: "*/5 * * * *"
+
+jobs:
+  notify:
+    # run slack-notifier workflow
+    uses: ./.github/workflows/call-slack-notifier.yml
+    with:
+      message: Starting scheduled Playwright e2e tests run...
+    secrets:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  test:
+    name: Run tests
+    uses: ./.github/workflows/call-run-tests.yml
+
+  publish_report:
+    name: Publish HTML Report
+    # using always() is not ideal here, because it would also run if the workflow was cancelled
+    if: "success() || needs.test.result == 'failure'"
+    needs: [test]
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    env:
+      REPO_URL: internetarchive.github.io/archiveorg-e2e-playwright
```

Comment:
Q: Is there a reason not to include the `https://` scheme in this env var too?

Reply:
will update it, thanks for catching it